### ISSUE TITLE
ROS support?

### ISSUE THREAD

**Issue by mfe7** [intel/gazebo-realsense#2]
Does this model support ROS? It would be great to subscribe to the image topics with my ROS node - any documentation on how to do this, or plans for future support? Thanks!

**Comment by rchiossi** [comment]
I don't believe we tested this with ROS and I'm unaware of the steps that would be required to do so. Our simulation was mostly done using our own bridge to the simulator.
Right now there are no plans to add support, but contributions are always welcome. :)

**Comment by stevegray954** [comment]
Someone has already wrapped this for ROS, though they based it on the version from August 2016.  https://github.com/SyrianSpock/realsense_gazebo_plugin
I see @mfe7 already has an issue open for it with Gazebo 9.

**Comment by awesome-manuel** [comment]
http://gazebosim.org/tutorials/?tut=ros_depth_camera
